contracts/eosiolib/action.h

```diff
@@ -70,7 +70,7 @@ extern "C" {
     *  Copy up to @ref len bytes of current action data to the specified location
     *  @brief Copy current action data to the specified location
     *  @param msg - a pointer where up to @ref len bytes of the current action data will be copied
-    *  @param len - len of the current action data to be copied
+    *  @param len - len of the current action data to be copied, 0 to report required size
     *  @return the number of bytes copied to msg
```

Comment:
Need something like ", or number of bytes that can be copied if len==0 passed", similarly for other return statements below.